### PR TITLE
April pipeline fixes, Workflow files, Added Cloud Support For Tasks 1.2.1 - 3

### PR DESCRIPTION
**Pull request by MrSteve81**
# Overall Review of Changes
April pipeline fixes, Workflow files, Added Cloud Support For Tasks 1.2.1 - 3

## Issue Fixes
N/A

## Enhancements
N/A

## How has this been tested?
Locally and using terraform pipelines in azure.

Please give an overview of how these changes were tested. If they were not please use N/A

### CODE REVIEW

File: .gitignore
Comment (uk-bolly) [inline .gitignore:1]: this needs to be ignored ideally so each add needs to be force updated. minimise potentail files in location that shouldn't be there

File: tasks/prelim.yml
Comment (uk-bolly) [inline tasks/prelim.yml:1]: do we want to add other cloud platforms? so we ensure we skip all cloud

File: .gitignore
Comment (MrSteve81) [reply .gitignore:1]: Fixed

File: .github/workflows/main.tf
[the remark this answers is not in the document]
Comment (MrSteve81) [reply .github/workflows/main.tf:1]: Added CIS to it for pipeline identifying where it originated for azure

**Review by georgenalen — APPROVED** [review]
looks good!

**Pull request merged**